spec/features/pages/form_name_question_spec.rb

```diff
@@ -25,5 +25,22 @@
         expect(page).to have_content 'Enter the form name or number, or select \'I don’t know the form name or number\''
       end
     end
+
+    context 'choosing the probate fees option' do
+      before do
+        Capybara.current_driver = :webkit
```

Comment:
so far this is the only spec using the capybara-webkit in our test suit so my thought was to avoid using it globally so to not increase the execution time.